src/ckeditor.jsx

```diff
@@ -97,7 +97,7 @@ class CKEditor extends React.Component {
 	}
 
 	_destroyEditor() {
-		this.editor.destroy();
+		this.editor && this.editor.destroy();
```

Comment:
It would be more readable using traditional `if` clause:

```suggestion
		if ( this.editor ) {
			this.editor.destroy();
		}
```